### PR TITLE
fix bq operator for table with google sheet

### PR DESCRIPTION
**Pull request by wapa5pow**
https://github.com/treasure-data/digdag/issues/1423

### CODE REVIEW

**Comment by yoyama** [comment]
@wapa5pow Thank you for your contribution and very sorry for late reply.
Currently we are working for major version upgrade. After then check this PR.

**Comment by yoyama** [comment]
`master` branch has been changed significantly. Could you change the PR based on current master?

**Comment by wapa5pow** [comment]
ok, done

File: digdag-standards/src/main/java/io/digdag/standards/operator/gcp/BqClient.java [hunk 1]
```diff
@@ -37,7 +40,10 @@
     protected Bigquery client(GoogleCredential credential, HttpTransport transport, JsonFactory jsonFactory)
     {
         if (credential.createScopedRequired()) {
-            credential = credential.createScoped(BigqueryScopes.all());
+            Set<String> scopes = new HashSet<String>();
+            scopes.addAll(BigqueryScopes.all());
+            scopes.add(SheetsScopes.DRIVE_READONLY);
```
Comment (yoyama) [inline digdag-standards/src/main/java/io/digdag/standards/operator/gcp/BqClient.java:45]: I am not sure google credential but I have a concern about adding scope affect to existence users or not.
It means that users need to change their credential in secrets or change permission on account even though they don't need to access to Google spreadsheet.
Could you explain about it ?

**Pull request closed without merging**